test/integration/model.test.js

```diff
@@ -1698,10 +1698,81 @@ describe(Support.getTestDialectTeaser('Model'), () => {
 
       await this.User.bulkCreate(data);
       let affectedRows = await this.User.destroy({ where: { secretValue: '42' } });
-      expect(affectedRows).to.equal(2);
+      expect(affectedRows).to.be.a('number').to.equal(2);
       affectedRows = await this.User.destroy({ where: { secretValue: '44' } });
-      expect(affectedRows).to.equal(0);
+      expect(affectedRows).to.be.a('number').to.equal(0);
     });
+    if (dialect === 'postgres') {
```

Comment:
use the Support table instead:

```suggestion
    if (this.sequelize.dialect.supports.delete.returning) {
```